lib/katello/plugin.rb

```diff
@@ -359,6 +359,14 @@
     :view_operatingsystems, :view_smart_proxies
   ]
 
+  role 'Content Importer', [
+    :import_content, :create_products, :create_content_views, :view_organizations
```

Comment:
can you also include `:edit_products`, `edit_content_views`  since the import needs to be able to update,